ansible/roles/xroad-ca/tasks/main.yml

```diff
@@ -1,18 +1,25 @@
----
-- name: check that OS != Redhat
+- name: show OS version
+  debug:
+    msg: "OS = {{ ansible_os_family }} "
+
+- name: check that OS = Redhat
   fail:
     msg: "This playbook only supports Ubuntu"
   when: ansible_os_family != "Debian"
 
 - name: ensure that acl utilities is installed
   # see https://docs.ansible.com/ansible/become.html#becoming-an-unprivileged-user
   package:
-    name: acl
-    state: present
+      name: acl
+      state: present
 
 - name: nginx-light
   apt: name=nginx-light state=present
 
+- name: instal incron
```

Comment:
installing incron is unnecessary (since the experimental ca web interface is not present)